common/dynamicconfig/static_properties.go

```diff
@@ -62,6 +62,11 @@ func GetBoolPropertyFnFilteredByNamespace(value bool) func(namespace string) boo
 	return func(namespace string) bool { return value }
 }
 
+// GetBoolPropertyFnFilteredByTaskQueueInfo returns value as BoolPropertyFnWithTaskQueueInfoFilters
+func GetBoolPropertyFnFilteredByTaskQueueInfo(value bool) func(namespace string, taskQueue string, taskType enumspb.TaskQueueType) bool {
```

Comment:
What I don't like about these helpers is that they tend to be used solely by their author.
We need some way to increase awareness of their existence.

I personally find it clearer to just inline the function and don't think that the couple of uses justify a helper.